### YamlMime:ManagedReference
items:
- uid: System.Xaml.XamlSchemaException
  id: XamlSchemaException
  children:
  - System.Xaml.XamlSchemaException.#ctor
  - System.Xaml.XamlSchemaException.#ctor(System.String)
  - System.Xaml.XamlSchemaException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Xaml.XamlSchemaException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: XamlSchemaException
  nameWithType: XamlSchemaException
  fullName: System.Xaml.XamlSchemaException
  type: Class
  summary: "Undantag som utlöses när en bindning system eller ett annat schema representation system för XAML rapporterar ett undantag i kontexten för schemat."
  remarks: "XamlSchemaException genereras av flera fall i CLR-bindning stöd klasser som innehåller schema-kontexten för .NET Framework XAML-tjänster (<xref:System.Xaml.XamlXmlReader>, <xref:System.Xaml.XamlObjectWriter>, eller <xref:System.Xaml.XamlServices>).</xref:System.Xaml.XamlServices> </xref:System.Xaml.XamlObjectWriter> </xref:System.Xaml.XamlXmlReader> En XamlSchemaException skickas vanligtvis som det inre undantaget för andra undantag som den <xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=fullName>-true-egenskapen för en <xref:System.Xaml.XamlObjectWriterException>, så det ska vara ovanligt att fånga en XamlSchemaException direkt om du implementerar en komponent, till exempel en skrivare särskilda objekt.</xref:System.Xaml.XamlObjectWriterException> </xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=fullName>"
  syntax:
    content: 'public class XamlSchemaException : System.Xaml.XamlException'
  inheritance:
  - System.Exception
  - System.Xaml.XamlException
  implements: []
  inheritedMembers:
  - System.Xaml.XamlException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Xaml.XamlException.LineNumber
  - System.Xaml.XamlException.LinePosition
  - System.Xaml.XamlException.Message
  platform:
  - net462
- uid: System.Xaml.XamlSchemaException.#ctor
  id: '#ctor'
  parent: System.Xaml.XamlSchemaException
  langs:
  - csharp
  name: XamlSchemaException()
  nameWithType: XamlSchemaException.XamlSchemaException()
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException()
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Initierar en ny instans av den <xref href=&quot;System.Xaml.XamlSchemaException&quot;> </xref> klass med ett meddelande som beskriver fel som tillhandahålls av systemet."
  syntax:
    content: public XamlSchemaException ();
    parameters: []
  overload: System.Xaml.XamlSchemaException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlSchemaException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Xaml.XamlSchemaException
  langs:
  - csharp
  name: XamlSchemaException(String)
  nameWithType: XamlSchemaException.XamlSchemaException(String)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(String)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Initierar en ny instans av den <xref href=&quot;System.Xaml.XamlSchemaException&quot;> </xref> klass med ett angivet meddelande som beskriver felet."
  syntax:
    content: public XamlSchemaException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Det meddelande som beskriver undantaget. Anroparen av den här konstruktorn är behövs för att säkerställa att den här strängen lokaliserade för den aktuella systemkulturen."
  overload: System.Xaml.XamlSchemaException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlSchemaException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Xaml.XamlSchemaException
  langs:
  - csharp
  name: XamlSchemaException(SerializationInfo,StreamingContext)
  nameWithType: XamlSchemaException.XamlSchemaException(SerializationInfo,StreamingContext)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Initierar en ny instans av den <xref href=&quot;System.Xaml.XamlSchemaException&quot;> </xref> klass med serialiserade data."
  syntax:
    content: protected XamlSchemaException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Det objekt som innehåller objektdata som serialiserade."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Detaljerad information om käll- och målservrar."
  overload: System.Xaml.XamlSchemaException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlSchemaException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Xaml.XamlSchemaException
  langs:
  - csharp
  name: XamlSchemaException(String,Exception)
  nameWithType: XamlSchemaException.XamlSchemaException(String,Exception)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(String,Exception)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Initierar en ny instans av den <xref href=&quot;System.Xaml.XamlSchemaException&quot;> </xref> klass med ett angivet meddelande och en referens till det interna undantaget som är orsaken till det här undantaget."
  remarks: "Ett undantag som utlöses när ett direkt resultat av ett tidigare undantag ska innehålla en referens till det tidigare undantaget i den <xref:System.Exception.InnerException%2A>egenskapen.</xref:System.Exception.InnerException%2A> Den <xref:System.Exception.InnerException%2A>-egenskap returnerar samma värde som överförs till konstruktorn eller `null` om den <xref:System.Exception.InnerException%2A>egenskapen tillhandahåller inte värdet i det interna undantaget för konstruktorn.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XamlSchemaException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Det meddelande som beskriver undantaget. Anroparen av den här konstruktorn är behövs för att säkerställa att den här strängen lokaliserade för den aktuella systemkulturen."
    - id: innerException
      type: System.Exception
      description: "Undantag som är orsaken till det aktuella undantaget. Om den `innerException` parametern är inte <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, det aktuella undantaget utlöses i en <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> block som hanterar det interna undantaget."
  overload: System.Xaml.XamlSchemaException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Xaml.XamlException
  isExternal: false
  name: System.Xaml.XamlException
- uid: System.Xaml.XamlSchemaException.#ctor
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException()
  nameWithType: XamlSchemaException.XamlSchemaException()
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException()
- uid: System.Xaml.XamlSchemaException.#ctor(System.String)
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException(String)
  nameWithType: XamlSchemaException.XamlSchemaException(String)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Xaml.XamlSchemaException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException(SerializationInfo,StreamingContext)
  nameWithType: XamlSchemaException.XamlSchemaException(SerializationInfo,StreamingContext)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Xaml.XamlSchemaException.#ctor(System.String,System.Exception)
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException(String,Exception)
  nameWithType: XamlSchemaException.XamlSchemaException(String,Exception)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Xaml.XamlSchemaException.#ctor*
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException
  nameWithType: XamlSchemaException.XamlSchemaException
